lib/routes/api/speech-credentials.js

```diff
@@ -110,24 +116,29 @@ router.get('/', async(req, res) => {
     res.status(200).json(creds.map((c) => {
       const {credential, ...obj} = c;
       if ('google' === obj.vendor) {
-        obj.service_key = JSON.parse(decrypt(credential));
+        const o = JSON.parse(decrypt(credential));
+        const obscured = {
+          ...o,
+          private_key: obscureKey(o.private_key)
```

Comment:
Looks like we need more work for just this one specifically. I think we'll have to "skip past" the initial part of the private key string, e.g. "-----BEGIN PRIVATE KEY-----\n" and leave the first digits after that static beginning comment in tact otherwise we end up with the following which isn't super useful lol

<img width="542" alt="Screen Shot 2022-08-22 at 9 54 12 AM" src="https://user-images.githubusercontent.com/438711/185976472-32f4b387-f375-4dde-8288-7069734cb267.png">

Reply:
Hope this works.

https://github.com/jambonz/jambonz-api-server/blob/e76c98ce9e62fcf99c25629f53497e44bb9da868/lib/routes/api/speech-credentials.js#L120-L124

Reply:
I am really tempted to use the `+` operator for these strings so it looks nicer.